docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx>=3.1.1
```

Comment:
Out of scope and is addressed by https://github.com/ansible/ansible-lint/pull/861